<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SANSEventSlicing</class>
 <widget class="QWidget" name="SANSEventSlicing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SANS Slicing Event Data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Run:</string>
       </property>
       <property name="buddy">
        <cstring>run_lineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="run_lineEdit">
       <property name="toolTip">
        <string>Insert here the run number or the file. You can select through the Scattering Sample widget.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="slicePushButton">
       <property name="toolTip">
        <string>Apply the slice at the data and update the values for the charge. It also updates the string for the event slicing. Press Enter</string>
       </property>
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="toolTip">
        <string>Close this widget. Press ESC</string>
       </property>
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_1">
           <property name="text">
            <string>Experiment duration: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="exp_t_time_label">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Total duration of the experiment in seconds.</string>
           </property>
           <property name="text">
            <string>time</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>[s]</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Slice From:</string>
           </property>
           <property name="buddy">
            <cstring>sliced_from_lineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sliced_from_lineEdit">
           <property name="toolTip">
            <string>Lower slicing limit. Default = 0.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>to</string>
           </property>
           <property name="buddy">
            <cstring>sliced_to_lineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sliced_to_lineEdit">
           <property name="toolTip">
            <string>Upper slice limit. Default = Max.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Charge</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="sliced_charge_label">
           <property name="toolTip">
            <string>Charge of the sliced data.</string>
           </property>
           <property name="text">
            <string>sliced</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>/</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="exp_charge_label">
           <property name="toolTip">
            <string>Charge of the whole experiment.</string>
           </property>
           <property name="text">
            <string>total</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>uC</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>run_lineEdit</tabstop>
  <tabstop>sliced_from_lineEdit</tabstop>
  <tabstop>sliced_to_lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SANSEventSlicing</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
